website/docs/intro/install/macos.mdx

```diff
@@ -0,0 +1,14 @@
+---
+sidebar_position: 5
+sidebar_label: MacOS
+description: |-
+  Install OpenTofu on MacOS.
+---
+
+# Installing OpenTofu on MacOS
+
+You can use OpenTofu as a [portable binary](portable.mdx) or you can install it using [Homebrew](https://brew.sh/). OpenTofu is available in the Homebrew Core repository, so you can install it by running:
```

Comment:
**(nitpick)** WDYT about linking the homebrew `tofu` page?

```suggestion
You can use OpenTofu as a [portable binary](portable.mdx) or you can install it using [Homebrew](https://formulae.brew.sh/formula/opentofu). OpenTofu is available in the Homebrew Core repository, so you can install it by running:
```

Reply:
Good catch, done!